aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie/Startup.cs

```diff
@@ -18,13 +16,17 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
+        #region ConfigureServices
```

Comment:
I recommend prefixing this region name with "snippet_" to make it obvious that it's used by a snippet import in the doc.